### ISSUE TITLE
Failure to execute image

### ISSUE THREAD

**Issue by Offia** [Qavah/qavah#1]
Please qavah.me I'm having problem on adding of image,on creating a campaign, it keeps telling me fail to execute  I should get image Data on canvas rendering context 2D the source width is 0 please I need assistance on please because I don't know how to go about it,

**Comment by yip-theodore** [comment]
Could you please try again from another device?

**Comment by Offia** [comment]
Qavah.me
<https://drive.google.com/file/d/1G0x9GXZeK46gcdTgS5Hfk0jcG9UqZ-hh/view?usp=drivesdk>
This is my campaign image, please help execute it so that I can create my
on campaign on qavah

**Comment by yip-theodore** [comment]
I'm sorry, the website doesn't support PDF files yet.
Do you have it in another format, please?

**Comment by Offia** [comment]
No,I have tried all the formart I have on my Android phone but it keeps
telling me the same thing(fail to execute'get dataimage on canvas
rendering,context2D width will be 0) please help me on the format which
site can support,🙏🙏🙏

**Comment by yip-theodore** [comment]
What about [this one](https://ibb.co/BPwXD99)? Also, does it work using your phone's camera directly?
